packages/vars/src/breakpoints.css

```diff
@@ -0,0 +1,15 @@
+/* If this is ever changed please update breakpoints.json as well */
+
+/* Mobile */
+@custom-media --mobile (max-width: 767px);
+
+/* Tablet */
+@custom-media --tablet (min-width: 768px);
+@custom-media --tablet-s (min-width: 768px);
+@custom-media --tablet-l (min-width: 1024px);
```

Comment:
don't we also have `--tablet-m` ?

Reply:
No, not in Figma or on the code that was on default.css before I moved to this new file